mysql-test/suite/rocksdb/t/compact_deletes.test

```diff
@@ -58,8 +58,8 @@ let $primary = 1;
 let $no_more_deletes = 0;
 --source suite/rocksdb/include/compact_deletes_test.inc
 
---exec echo Test 4: Do a bunch of secondary key updates and set the compaction sysvar
---exec echo Expect: compaction
+--echo Test 4: Do a bunch of secondary key updates and set the compaction sysvar
+--echo Expect: compaction
```

Comment:
This can probably be upstreamed.  I'll make a note to do it in the next test fixup push.

Reply:
Everything except force-rmdir can be upstreamed to 5.6